mitodl/mitxpro
# refactor!: rename visible_in_bulk_form to is_private in product model

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/mitxpro/issues/2697

#### What's this PR do?
Renames Product.visible_in_bulk_form to is_private. Also, inverts & migrates the data to the is_private field.

#### How should this be manually tested?

- Checkout master branch
- Create some products with visible_in_bulk_form = True and some with visible_in_bulk_form = False.
- Checkout this branch and run the migrations.
- Note that the visible_in_bulk_form is removed and is_private is added. Also, is_private contains the inverted data from the visible_in_bulk_form.
- Verify that the bulk coupon creation has the same list of products as on master branch. Also, there are no changes in the bulk coupon form.
